### PR TITLE
[13.0][MIG] report_batch

### PR DESCRIPTION
**Pull request by bodedra**
Supersede https://github.com/OCA/reporting-engine/pull/369

### CODE REVIEW

**Comment by bodedra** [comment]
@kittiu @dreispt @max3903

**Review by dreispt — APPROVED** [review]

File: report_batch/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Batch Report Printing",
+    "summary": "Ability to print multiple QWeb reports in a single batch.",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "Open Source Integrators, Odoo Community Association (OCA)",
+    "category": "Reporting",
+    "website": "http://www.opensourceintegrators.com",
```
Comment (Saran440) [inline report_batch/__manifest__.py]:
```suggestion
    "website": "https://github.com/OCA/reporting-engine",
```

**Review by Saran440 — COMMENTED** [review]
ROADMAP.rst is not necessary (if not have, please delete it.)

Functional Test.
I config following:
Selection_002

But I can't open pdf generate by batch report. Other report can open.
Selection_003

**Comment by bodedra** [comment]
@Saran440 I have tested on runbot and created functional video. It works fine.

http://3417887-386-928d12.runbot2-2.odoo-community.org/web?debug=1#action=187&model=stock.picking&view_type=list&cids=&menu_id=93

https://youtu.be/QSRev9Tac4Q

**Review by Chandresh-SerpentCS — APPROVED** [review]

**Comment by bodedra** [comment]
@Saran440 Would you please update your review?

**Comment by Saran440** [comment]
@bodedra yes, It's works find when I tested on runbot. but I tested by create new database on localhost it's error. my code is up-to-date.

1 subreport   -> I can open pdf.
2 subreport+ -> I can't open pdf.

**Comment by bodedra** [comment]
@Saran440 I am not sure about your localhost. It works well here.

**Comment by bodedra** [comment]
@Saran440 Based on your runbot test, can you approve this PR?

**Review by Saran440 — APPROVED** [review]

**Comment by dreispt** [comment]
\ocabot merge patch

**Pull request merged**